installing/installing_azure/installing-azure-private.adoc

```diff
@@ -89,6 +89,13 @@ include::modules/cco-ccoctl-creating-at-once.adoc[leveloffset=+3]
 // Additional steps for the Cloud Credential Operator utility (`ccoctl`)
 include::modules/cco-ccoctl-install-creating-manifests.adoc[leveloffset=+3]
 
+include::modules/installing-private-image-registry-private-azure.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* See xref:../../installing/installing_azure/installing-azure-account.adoc#minimum-required-permissions-ipi-azure_installing-azure-account[Required Azure permissions for installer-provisioned infrastructure] for the list of permissions needed to create a private storage endpoint.
```

Comment:
Rewrite as "For the list of permissions needed to create a private storage endpoint, see ...".